context/discovery-ontology.ttl

```diff
@@ -14,7 +14,10 @@
 : rdf:type owl:Ontology ;
 	dc:creator "Andrea Cimmino" ;
 	dc:creator "Farshid Tavakolizadeh" ;
-	owl:versionInfo "0.1.2" ;
+	owl:versionInfo "1.0.1" ;

```

Comment:
```suggestion
	owl:versionInfo "1.0.0-a2" ;

```

Reply:
I've applied the new version now and also adjusted the `priorVersion` URL so that it actually fetches the `.ttl` file from GitHub.